sql/moz-fx-data-shared-prod/telemetry/unified_metrics/view.sql

```diff
@@ -2,6 +2,6 @@ CREATE OR REPLACE VIEW
   `moz-fx-data-shared-prod.telemetry.unified_metrics`
 AS
 SELECT
-  * REPLACE (COALESCE(country, '??') AS country)
+  * REPLACE (COALESCE(country, 'UNDEFINED') AS country)
```

Comment:
This is to unify the value for not-known values.

Reply:
`'UNDEFINED'` won't map to any rows in `country_codes_v1`. `'??'` is the "ISO code" representing unknown countries and maps to this row when joined:
```
name | code | region_name | subregion_name | intermediate_region_name
Unknown Country Name | ?? | Unknown Region | Unknown Sub-region | Unknown Intermediate Region
```